components/properties/src/sets.rs

```diff
@@ -1824,6 +1835,202 @@ pub fn load_for_general_category_group(
+        }
+        UnicodeProperty::Extender => load_set_data::<ExtenderV1Marker, _>(provider),
+        UnicodeProperty::GraphemeBase => load_set_data::<GraphemeBaseV1Marker, _>(provider),
+        UnicodeProperty::GraphemeExtend => load_set_data::<GraphemeExtendV1Marker, _>(provider),
+        UnicodeProperty::HexDigit => load_set_data::<HexDigitV1Marker, _>(provider),
+        UnicodeProperty::IdsBinaryOperator => {
+            load_set_data::<IdsBinaryOperatorV1Marker, _>(provider)
+        }
+        UnicodeProperty::IdsTrinaryOperator => {
+            load_set_data::<IdsTrinaryOperatorV1Marker, _>(provider)
+        }
+        UnicodeProperty::IdContinue => load_set_data::<IdContinueV1Marker, _>(provider),
+        UnicodeProperty::IdStart => load_set_data::<IdStartV1Marker, _>(provider),
+        UnicodeProperty::Ideographic => load_set_data::<IdeographicV1Marker, _>(provider),
+        UnicodeProperty::JoinControl => load_set_data::<JoinControlV1Marker, _>(provider),
+        UnicodeProperty::LogicalOrderException => {
+            load_set_data::<LogicalOrderExceptionV1Marker, _>(provider)
+        }
+        UnicodeProperty::Lowercase => load_set_data::<LowercaseV1Marker, _>(provider),
+        UnicodeProperty::Math => load_set_data::<MathV1Marker, _>(provider),
+        UnicodeProperty::NoncharacterCodePoint => {
+            load_set_data::<NoncharacterCodePointV1Marker, _>(provider)
+        }
+        UnicodeProperty::PatternSyntax => load_set_data::<PatternSyntaxV1Marker, _>(provider),
+        UnicodeProperty::PatternWhiteSpace => {
+            load_set_data::<PatternWhiteSpaceV1Marker, _>(provider)
+        }
+        UnicodeProperty::QuotationMark => load_set_data::<QuotationMarkV1Marker, _>(provider),
+        UnicodeProperty::Radical => load_set_data::<RadicalV1Marker, _>(provider),
+        UnicodeProperty::RegionalIndicator => {
+            load_set_data::<RegionalIndicatorV1Marker, _>(provider)
+        }
+        UnicodeProperty::SentenceTerminal => load_set_data::<SentenceTerminalV1Marker, _>(provider),
+        UnicodeProperty::SoftDotted => load_set_data::<SoftDottedV1Marker, _>(provider),
+        UnicodeProperty::TerminalPunctuation => {
+            load_set_data::<TerminalPunctuationV1Marker, _>(provider)
+        }
+        UnicodeProperty::UnifiedIdeograph => load_set_data::<UnifiedIdeographV1Marker, _>(provider),
+        UnicodeProperty::Uppercase => load_set_data::<UppercaseV1Marker, _>(provider),
+        UnicodeProperty::VariationSelector => {
+            load_set_data::<VariationSelectorV1Marker, _>(provider)
+        }
+        UnicodeProperty::WhiteSpace => load_set_data::<WhiteSpaceV1Marker, _>(provider),
+        UnicodeProperty::XidContinue => load_set_data::<XidContinueV1Marker, _>(provider),
+        UnicodeProperty::XidStart => load_set_data::<XidStartV1Marker, _>(provider),
+        _ => Err(PropertiesError::unexpected_property(name)),
```

Comment:
Nit: This should be unreachable. Can you debug-assert that?

Reply:
No, it's not, it might parse as gc or sc or scx